pkg/api/types.go

```diff
@@ -753,6 +754,16 @@ func (p *Properties) getAgentPoolIndexByName(name string) int {
 	return index
 }
 
+// GetPrefix returns resource prefix based on Properties
+func (p *Properties) GetPrefix() (ret string) {
```

Comment:
GetPrefix is too vague, let's have something specific. `GetClusterPrefix` would work IMO. Also open to other names :)

Reply:
`GetClusterName` ?